index.yaml

```diff
@@ -3,13 +3,13 @@ indexes:
 # AUTOGENERATED
 
 # This index.yaml is automatically updated whenever the dev_appserver
-# detects that a new type of query is run.  If you want to manage the
-# index.yaml file manually, remove the above marker line (the line
-# saying "# AUTOGENERATED").  If you want to manage some indexes
-# manually, move them above the marker line.  The index.yaml file is
-# automatically uploaded to the admin console when you next deploy
-# your application using appcfg.py.
-
+ # detects that a new type of query is run.  If you want to manage the
```

Comment:
Just to be clear, it's fine to drop this comment if that's what happened automatically (otherwise it'll just end up getting deleted again anyway).

If you do want to keep it, note that you seem to have added a space at the front of each line...

Reply:
Removed the comment as it was removed automatically.